src/main/java/com/conveyal/r5/analyst/cluster/WorkerStatus.java

```diff
@@ -24,6 +24,8 @@
 public class WorkerStatus {
 
     private static final Logger LOG = LoggerFactory.getLogger(WorkerStatus.class);
+    private static final int LEGACY_WORKER_MAX_TASKS = 16;
+    public static final int LEGACY_WORKER_MAX_POLL_INTERVAL_SECONDS = 2 * 60;
```

Comment:
Right, when the workers poll, they send a value `pollIntervalSeconds` which is the maximum amount of time until the worker will poll again. This is not hard-wired in case we need to change the value in future workers, or even dynamically adjust it on a single worker. If the backend is not polled by that same worker within `pollIntervalSeconds` (plus some safety factor to account for contention or requests piling up) the worker must have shut down or become otherwise incapacitated. In the past, the workers did not supply this value, and the backend would assume any worker was shut down if it didn't poll every `WORKER_RECORD_DURATION_MSEC` (creating problems when workers were too busy crunching numbers for difficult tasks).